src/components/app-content-renderer/recommendation-tile.js

```diff
@@ -23,7 +22,7 @@ import { useIntl } from 'react-intl';
 
 import useRequest from './use-request';
 
-const NoIcon = () => <span>No icon</span>;
+const NoIcon = () => <span className="no-icon">No icon</span>;
```

Comment:
Can you name this class `ins-c-no-icon`? So we are aligned with the rest of the classnames?